### PR TITLE
Add CRUD Index page to the new Admin Panel 

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | bootstrap-admin-panel (2.0)
| Bug fix?        | no
| New feature?    | 
| BC breaks?      | 
| Deprecations?   | 
| Related tickets | based on #15352
| License         | MIT

CleanShot 2023-09-23 at 17 27 45@2x

CleanShot 2023-09-23 at 17 28 26@2x

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/config/app/template_events.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# This file is part of the Sylius package.
+# (c) Sylius Sp. z o.o.
+
+imports:
+    - { resource: 'template_events/*.yaml' }
+
+sylius_ui:
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Resources/config/app/template_events.yaml:7]: Different names for `syiius_ui` bundle and `template_events` files will be a source of confusion.

File: src/Sylius/Bundle/AdminBundle/Resources/assets/entrypoint.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Resources/assets/entrypoint.js:1]: Random question, will bundles for 2.0 be converted to SF dir structure?

File: src/Sylius/Bundle/AdminBundle/Resources/config/app/template_events.yaml (hunk 1 above)
Comment (jakubtobiasz) [reply src/Sylius/Bundle/AdminBundle/Resources/config/app/template_events.yaml:7]: we'll unify that at some point, we'll go with `ui events` or `template events` as we have `sylius_template_event` and `sylius_ui.events` 🤷🏼‍♂️.

File: src/Sylius/Bundle/AdminBundle/Resources/assets/entrypoint.js (hunk 2 above)
Comment (jakubtobiasz) [reply src/Sylius/Bundle/AdminBundle/Resources/assets/entrypoint.js:1]: Yes, but we'll do it as one of the last tasks. Doing it right now will cause many conflicts while upmerging 1.13 -> 2.0.

File: src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/Common/_content.html.twig [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{% set event_name = event_prefix ~ '.content' %}
+{% set resource_event_name = resource_event_prefix ~ '.content' %}
+
+<div class="page-wrapper">
+    {{ sylius_template_event([resource_event_name, event_name], _context|merge({ event_prefix: event_name, resource_event_prefix: resource_event_name })) }}
```
Comment (loic425) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/Common/_content.html.twig:5]:
```suggestion
    {{ sylius_template_event([resource_event_name, event_name, 'sylius.admin.index.body.content'], _context|merge({ event_prefix: event_name, resource_event_prefix: resource_event_name })) }}
```

**Comment by jakubtobiasz** [comment]
Closed in favor of #15361

**Pull request closed without merging**